src/modules/colorPicker/ColorPickerUI/Views/ColorEditorView.xaml

```diff
@@ -274,7 +292,8 @@
                 TextWrapping="Wrap" />
         </StackPanel>
 
-        <Border
+        <Border 
+            Background="{DynamicResource {x:Static SystemColors.AccentColorBrushKey}}"
```

Comment:
```suggestion
            BorderBrush="{DynamicResource SurfaceStrokeColorFlyoutBrush}"
```

Reply:
Do you mean to replace BorderBrush to SurfaceStrokeColorFlyoutBrush and to remove the Background? Or just replace the BorderBrush?